app/frontend/src/features/auth/Logout.tsx

```diff
@@ -17,6 +18,8 @@ export default function Logout() {
       queryClient.resetQueries("hostRequestMessages");
       queryClient.resetQueries("hostRequests");
       queryClient.resetQueries("ping");
+      queryClient.resetQueries(friendRequestKey("Sent"));
+      queryClient.resetQueries(friendRequestKey("Received"));
```

Comment:
Yeah that might make this more maintainable going forward tbh, otherwise it's easy to forget to remove a query here. I guess there's probably still value in having the cache while the user is logged in so that between page refreshes it'll still be faster than no cache